notebooks/official/experiments/get_started_with_vertex_experiments.ipynb

```diff
@@ -0,0 +1,1581 @@
+{
```

Comment:
<p>Is there a reason this notebook is not using the latest notebook template that can significantly reduce the boilerplate? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
n/a